tests/sentry/deletions/test_group.py

```diff
@@ -3,10 +3,10 @@
 from uuid import uuid4
 
 from sentry import nodestore
-from sentry.deletions.defaults.group import EventDataDeletionTask
+from sentry.deletions.defaults.group import ErrorEventsDeletionTask
 from sentry.deletions.tasks.groups import delete_groups
 from sentry.eventstore.models import Event
-from sentry.issues.grouptype import ReplayDeadClickType
+from sentry.issues.grouptype import FeedbackGroup
```

Comment:
The first use case for deletion is going to be User Feedback. Either group type would have worked. This is not even important to change.